javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/matcher/AdditionalLibraryIgnoresMatcher.java

```diff
@@ -42,7 +42,6 @@ public boolean matches(T target) {
         || name.startsWith("com.jayway.jsonpath.")
         || name.startsWith("com.lightbend.lagom.")
         || name.startsWith("javax.el.")
-        || name.startsWith("net.sf.cglib.")
```

Comment:
why?

Reply:
this was a mistake, just pushed it back 👍